hermit/shamir_share.py

```diff
@@ -32,9 +32,19 @@
 import hmac
 import os
 
+import shamir_mnemonic
 from shamir_mnemonic import *
```

Comment:
I dislike this pattern, either use the fact that `RANDOM_BYTES` are already in this namespace, or move the rest of `shamir_mnemonic`s imports. As is, there is a `hermit.shamir_share.RANDOM_BYTES` and a `hermit.shamir_share.shamir_mnemonic.RANDOM_BYTES`. You know my preference is to get rid of the `import *` ;)

Reply:
Actually, I'm not sure how this works. but it's confusing

Reply:
I agree, but it is the way that the upstream library implemented changing out the random number generator.

The methods in shamir_mnemonic use the RANDOM_BYTES method referenced in that module, so if you want to change it out, you have to modify the value in that namespace. If I just import RANDOM_BYTES into this namespace, changing its value here does not change the value used by the imported methods. 

If we were not extending the upstream library to post-encrypt the shards, we would not have to deal with this complication, but we are and we are doing it in such a way that the rest of the hermit code uses this file in lieu of the upstream module to do all of its sharding operations.

Perhaps some comments in the code are in order to explain the insanity.